libraries/aws_cloudfront_distribution.rb

```diff
@@ -68,6 +68,17 @@ def initialize(opts = {})
       end
     end
     @custom_origin_ssl_protocols = @custom_origin_ssl_protocols.uniq.sort
+
+    # Find aws cloudfront distribution origin path.
+    # Either return path string, or ""
+    return unless opts[:origin_domain_name]
+    @origin_domain_name = opts[:origin_domain_name]
+    @s3_origin_path = ''
+    config.origins.items.each do |origin_config|
+      if origin_config.domain_name == @origin_domain_name
```

Comment:
why not collect all origin paths ?

Reply:
The idea of this feature is to validate a specific `origin path` for  given `origin_domain_name` of specific CF distro
if we want to validate all the `origin_path` either it has to be empty or same path for all the domain name.
Or I am missing something ? let me know your thoughts